### ISSUE TITLE
Radio Buttons in k-filter-menu-container Misaligned

### ISSUE THREAD

**Issue by DaleyKD** [telerik/kendo-themes#2540]
**Describe the bug**
After updating from `4.16.1` to `4.35.1` (and `2020.2.513` to `2021.2.511`), a column filter that uses only two radio buttons now is misaligned.

**To reproduce**
Snippet for setting up the column:
``` js
{ field: "isActive", title: "@Html.DisplayNameFor(m => m.ViewModel.AccountStatus)", template: "#: accountStatus #", filterable: { messages: { isTrue: "@StratusStrings.Enabled", isFalse: "@StratusStrings.Disabled" } } },
```

**Expected behavior**
Nothing should have changed (except button styles?)

**Screenshots**
New
![image](https://user-images.githubusercontent.com/7206282/121425631-418c5700-c938-11eb-94dc-3089386a50b7.png)

Old
![image](https://user-images.githubusercontent.com/7206282/121426434-3259d900-c939-11eb-99a0-1032549f4f8d.png)

**Affected package (please remove the unneeded items)**
* theme-bootstrap v4.35.1

**Affected suites (please remove the unneeded items)**
* Kendo UI for jQuery 2021.2.511

**Affected browsers (please remove the unneeded items)**
* All _(I assume)_

**Other info**
New HTML
``` html
<form title="Show items with value that:" class="k-filter-menu k-popup k-group k-reset k-state-border-up" data-role="popup" aria-hidden="false" style="position: absolute; display: flex; transform: translateY(0px);">
	<div class="k-filter-menu-container">
		<div class="k-filter-help-text">Show items with value that:</div>
		<label>
			<input type="radio" data-bind="checked: filters[0].value" value="true" name="filters[0].value">Enabled</label>
		<label>
			<input type="radio" data-bind="checked: filters[0].value" value="false" name="filters[0].value">Disabled</label>
		<div class="k-action-buttons">
			<button type="submit" title="Filter" class="k-button k-primary">Filter</button>
			<button type="reset" title="Clear" class="k-button">Clear</button>
		</div>
	</div>
</form>
```

Old HTML
``` html
<form title="Show items with value that:" class="k-filter-menu k-popup k-group k-reset k-state-border-up" data-role="popup" aria-hidden="false" style="position: absolute; font-size: 12.96px; font-family: Lato, &quot;Helvetica Neue&quot;, Arial, sans-serif; font-stretch: 100%; font-style: normal; font-weight: 400; line-height: 19.44px; display: flex; transform: translateY(0px);">
	<div class="k-filter-menu-container">
		<div class="k-filter-help-text">Show items with value that:</div>
		<label>
			<input type="radio" data-bind="checked: filters[0].value" value="true" name="filters[0].value">Enabled</label>
		<label>
			<input type="radio" data-bind="checked: filters[0].value" value="false" name="filters[0].value">Disabled</label>
		<div class="k-action-buttons">
			<button type="submit" title="Filter" class="k-button k-primary">Filter</button>
			<button type="reset" title="Clear" class="k-button">Clear</button>
		</div>
	</div>
</form>
```

**Comment by DaleyKD** [comment]
And update?

**Comment by joneff** [comment]
This will be fixed with 2021 R3. It's part theme, part javascript.

This is kendo-jquery, correct?

**Comment by DaleyKD** [comment]
@joneff : Yeah, I'm in kendo-jquery for this.  I haven't double-checked it in my kendo-react.

What's the approx. expected release date for R3?  (I had planned on upgrading to R2 this next Programming Increment, but will postpone if R3 is soon-ish.)

**Comment by joneff** [comment]
September 15th, which, I know, is about 6 weeks from now.

**Comment by epetrow** [comment]
This will be fixed with the next official version: fd954fefeb601aa1f7cd7f2b27183302f29ac09c
